app/services/mapping/declaration/PresentationOfficeBuilder.scala

```diff
@@ -26,9 +26,8 @@ import wco.datamodel.wco.declaration_ds.dms._2._
 
 object PresentationOfficeBuilder {
 
-  def build(implicit cacheMap: CacheMap): Declaration.PresentationOffice =
-    cacheMap
-      .getEntry[Choice](Choice.choiceId)
+  def build(implicit cacheMap: CacheMap, choice: Choice): Declaration.PresentationOffice =
+    Option(choice)
```

Comment:
Similar situation here, why you're wrapping Choice into Option?

Reply:
Thanks! Please review again! :)

Reply:
I have addressed this! :)